--- startup_stm32l496xx.s	2025-03-05 15:22:48.963133633 +0400
+++ startup_stm32l496xx.s.fix	2025-03-05 15:28:15.227134486 +0400
@@ -3,27 +3,8 @@
   * @file      startup_stm32l496xx.s
   * @author    MCD Application Team
   * @brief     STM32L496xx devices vector table GCC toolchain.
-  *            This module performs:
-  *                - Set the initial SP
-  *                - Set the initial PC == Reset_Handler,
-  *                - Set the vector table entries with the exceptions ISR address,
-  *                - Configure the clock system  
-  *                - Branches to main in the C library (which eventually
-  *                  calls main()).
-  *            After Reset the Cortex-M4 processor is in Thread mode,
-  *            priority is Privileged, and the Stack is set to Main.
   ******************************************************************************
-  * @attention
-  *
-  * Copyright (c) 2017 STMicroelectronics.
-  * All rights reserved.
-  *
-  * This software is licensed under terms that can be found in the LICENSE file
-  * in the root directory of this software component.
-  * If no LICENSE file comes with this software, it is provided AS-IS.
-  *
-  ******************************************************************************
-  */
+*/
 
   .syntax unified
 	.cpu cortex-m4
@@ -544,4 +525,3 @@
 	
 	.weak	DMA2D_IRQHandler
 	.thumb_set FPU_IRQHandler,Default_Handler    
-
